### ISSUE TITLE
When exporting a polygon the export crashes (1)

### ISSUE THREAD

**Issue by SaturnHafen** [simulate-digital-rail/demo#6]
Hello from the bachelors project :)

When selecting this polygon `52.425454033640584 13.101882934570314 52.42503531994297 13.222732543945312 52.363860434566206 13.211059570312502 52.37434160350472 13.100509643554688` and trying to export it to a planpro file, the export crashes with a 500 internal server error.

Server backtrace:
```
127.0.0.1 - - [19/Jan/2023 10:52:46] "GET /run?polygon=52.425454033640584+13.101882934570314+52.42503531994297+13.222732543945312+52.363860434566206+13.211059570312502+52.37434160350472+13.100509643554688&mode=planpro HTTP/1.1" 500 -
Error on request:
Traceback (most recent call last):
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/werkzeug/serving.py", line 335, in run_wsgi
    execute(self.server.app)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/werkzeug/serving.py", line 322, in execute
    application_iter = app(environ, start_response)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 2548, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.handle_exception(e)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/saturnhafen/Code/Bachelorprojekt/yaramo-testing/demo/app.py", line 19, in run_converter
    topology = ORMImporter().run(polygon)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/orm_importer/importer.py", line 136, in run
    self._add_signals(path, current_edge, node, next_top_node)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/orm_importer/importer.py", line 85, in _add_signals
    direction=getSignalDirection(edge, self.ways, path, node.tags["railway:signal:direction"]),
KeyError: 'railway:signal:direction'
```

**Comment by jeriox** [comment]
Hey there :)

Are you using the most recent version of the importer? I stumbled upon the exact same error and fixed it in https://github.com/simulate-digital-rail/orm-importer/commit/f26e3ccf344cf530959588924df540de5cd406fd#diff-0ecf1c43c93f717cb284251f89b32e3a44cf73c0896810e5f12489a11b2a3ae0R79

**Comment by ctiedt** [comment]
Should be fixed by #5. Please re-open this issue if the problems persist.
